test/lib/analyzer/image-inspector.spec.ts

```diff
@@ -0,0 +1,265 @@
+import * as fs from "fs";
+import * as path from "path";
+import * as tmp from "tmp";
+import { v4 as uuidv4 } from "uuid";
+
+import * as imageInspector from "../../../lib/analyzer/image-inspector";
+import { ArchiveResult } from "../../../lib/analyzer/types";
+import { Docker } from "../../../lib/docker";
+import * as subProcess from "../../../lib/sub-process";
+
+function rmdirRecursive(customPath: string[]): void {
+  if (customPath.length < 2) {
+    return;
+  }
+
+  fs.rmdirSync(path.join(...customPath));
+  const next = customPath.slice(0, customPath.length - 1);
+  rmdirRecursive(next);
+}
+
+// prettier-ignore
```

Comment:
Why so? 😄

Reply:
Prettier would break the table formatting on the "it.each" block, making it hard to read :) I learnt that this comment only makes Prettier ignores the block immediately after it, so Prettier starts up again after this `describe`.